cogment_verse/processes/model_registry.py

```diff
@@ -30,11 +30,11 @@ def create_model_registry_service(work_dir, model_registry_cfg, services_directo
     )
 
     return CogmentCliProcess(
-        name="model_registry",
+        name=ServiceType.MODEL_REGISTRY.value,
         work_dir=work_dir,
         cli_args=[
             "services",
-            "model_registry",
+            ServiceType.MODEL_REGISTRY.value,
```

Comment:
It's just a coincidence that the CLI arg is the same than the service type value